core/src/main/scala/org/genivi/sota/core/BlacklistResource.scala

```diff
@@ -42,6 +42,11 @@ class BlacklistResource(namespaceExtractor: Directive1[AuthedNamespaceScope],
         bl <- BlacklistedPackages.create(namespace, req.packageId, req.comment)
         _ <- messageBus.publishSafe(PackageBlacklisted(namespace, req.packageId, Instant.now()))
         _ <- db.run(UpdateSpecs.cancelAllUpdatesByStatus(UpdateStatus.Pending, namespace, req.packageId))
+        _ <- db.run(UpdateSpecs.findByPackageId(namespace, req.packageId)).map { res =>
```

Comment:
It can since `cancelAllUpdatesByStatus` is only called from here. OTOH `cancelAllUpdatesByStatus` is in a DB object to which I didn't necessarily wanted to add a message bus variable and the function would then need a logic to go from one status to the next. How about creating a helper method in `BlacklistResource`?